fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -833,6 +900,26 @@ void DLApi::init() {
 	                   fdbCPath,
 	                   "fdb_tenant_wait_purge_granules_complete",
 	                   headerVersion >= ApiVersion::withBlobRangeApi().version());
+	loadClientFunction(&api->tenantBlobbifyRange,
```

Comment:
The tricky part is they will be in different snowflake releases, but on main, they're both new since apple/7.1.
So that's probably something we'll have to handle specially on the release branch, i'd assume

Reply:
Then we can introduce separate flag like withBlobRangeApi2() or withTenantBlobRangeApi(), which then will be assigned to different versions in our release branch

Reply:
Makes sense, let's do that

Reply:
Done.